Firmware/ultralcd.cpp

```diff
@@ -676,6 +676,9 @@ void lcdui_print_status_line(void)
         case CustomMsg::Resuming: //Resuming
             lcd_puts_at_P(0, 3, _T(MSG_RESUMING_PRINT));
             break;
+        case CustomMsg::M117:
```

Comment:
Merge this case with these:
```
        case CustomMsg::MsgUpdate: //Short message even while printing from SD
        case CustomMsg::Status: // Nothing special, print status message normally
        case CustomMsg::M0Wait: // M0/M1 Wait command working even from SD
```
They are all identical and could probably be optimized better this way.

Reply:
Done